content/source/docs/enterprise/before-installing/cluster-architecture.html.md

```diff
@@ -25,6 +25,16 @@ Clustered deployment relies on a Terraform module to provision infrastructure. C
 
 A Terraform Enterprise cluster relies on a load balancer to direct traffic to application instances.
 
+## Capacity Management
+
+Important aspect to the cluster is how resources are used to execute Terraform runs because this has a direct impact upon how much work a cluster can to concurrently.
+
+By default, each Terraform run is allocated 512MB of RAM which has the majority of the impact upon the ability to schedule new Terraform runs.
+
+To calculate how big a cluster should be, a customer needs to first decide how many concurrent Terraform runs they require. For instance, if they need to be able to execute 100 runs simultaniously, then the cluster needs to have at least 50GB of RAM available for runs. Because the services that run the Terraform Enterprise application run also share that memory, we suggest adding 16GB to the number. So in this case, the cluster would require 66GB of RAM. **The most common way to achieve at least 66GB of cluster RAM is a pool of 5 secondary instances and 3 primary instances, all with 16GB of RAM**.
```

Comment:
```suggestion
To calculate how big a cluster should be, first decide how many concurrent Terraform runs you require. For instance, if you need to be able to execute 100 runs simultaneously, then the cluster needs at least 50GB of RAM available for runs. Because the services that run the Terraform Enterprise application run also share that memory, we suggest adding 16GB to the number. So in this case, the cluster would require 66GB of RAM. The most common way to achieve at least 66GB of cluster RAM is a pool of 5 secondary instances and 3 primary instances, all with 16GB of RAM.
```